types/serverless/plugins/aws/provider/awsProvider.d.ts

```diff
@@ -136,7 +136,7 @@ declare namespace Aws {
     interface JwtAuthorizer {
         identitySource: string;
         issuerUrl: string;
-        audience: string[];
+        audience: string | string[];
```

Comment:
Cloudformation intrinsic functions are allowed as well, can you add those definition of at least allow the usage fo `Record<string, unknown> | Record<string, unknown>[]` as well ?